otter_welcome_buddy/common/constants.py

```diff
@@ -5,6 +5,7 @@ class CronExpressions(Enum):
     """Defined Cron Expressions"""
 
     DAY_ONE_OF_EACH_MONTH_CRON: str = "0 0 1 * *"
+    NON_ENGLISH_TEACHERS_MESSAGE_TIME: str = "0 13 * * 2"  # 12:00 PM Pacific Time
```

Comment:
_12:00 PM Pacific Time_ that depends if is Summer time or not, I suggest you change to this:

```suggestion
    TUESDAY_NOON_CRON: str = "0 12 * * 2"
```